GDJS/GDJS/Events/CodeGeneration/EventsCodeGenerator.h

```diff
@@ -65,6 +67,7 @@ class EventsCodeGenerator : public gd::EventsCodeGenerator {
    */
   static gd::String GenerateEventsFunctionCode(
       gd::Project& project,
+      const gd::EventsFunctionsContainer functionContainer,
```

Comment:
Pass it by reference?
```suggestion
      const gd::EventsFunctionsContainer& functionsContainer,
```

Otherwise we'll pay for a copy every time and this can quickly adds up in terms of performance.